web-components/src/components/icons/ProfileIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { sxToArray } from '../../utils/mui/sxToArray';
+
+interface ProfileIconProps extends SvgIconProps {}
+
+export default function ProfileIcon({
+  sx,
+  ...props
+}: ProfileIconProps): JSX.Element {
+  return (
+    <SvgIcon
+      sx={[{ color: '#4FB573' }, ...sxToArray(sx)]}
```

Comment:
It's best not to set fixed colors for SVG to make them more reusable.
We can give this one as a param.

Reply:
ok, that makes sense, i think i did this right, i basically just replaced:
```
{ color: '#4FB573' }, ...sxToArray(sx)
```
with:
```
...sxToArray(sx)
```
and then changed the `ProfileIcon` call site to:
```
<ProfileIcon sx={{ ml: 2, cursor: 'pointer', color: '#4FB573' }}/>
```

seems to be working ok, color looks the same

Reply:
the icon looks blank on storybook 
![image](https://github.com/regen-network/regen-web/assets/3149358/d3c88666-ab08-4f6d-92d8-3be6a7551176)

we should probably specify some color in the story or default color in the component
should we also rename this to `ShareIcon`? I guess it could be used elsewhere than on profile pages.

Reply:
whoops, forgot about the storybook, will fix that. and that's a better name, i'll make that change

Reply:
added a color default in the component here:
aac423ed5976c4fdafb207091d4f663c76e169dd

Reply:
renamed to ShareIcon here:
a65efd1de4da95f06210b2620f8b55db4f9148bf